entries/a/airtable.com.json

```diff
@@ -6,7 +6,8 @@
       "call",
       "totp"
     ],
-    "documentation": "https://support.airtable.com/hc/en-us/articles/219409917",
+    "documentation": "https://support.airtable.com/docs/enabling-two-factor-authentication",
+    "recovery": "https://support.airtable.com/v1/docs/contacting-airtable-support",
```

Comment:
Maybe link to the FAQ section of the 2FA documentation instead, since it provides some context?

Reply:
Could you suggest the URL? :)

Reply:
```suggestion
    "recovery": "https://support.airtable.com/docs/enabling-two-factor-authentication#faqs",
```